101-cosmosdb-free/azuredeploy.parameters.us.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
remove this file entirely since the defaults should work for Gov

Reply:
So this will run for US Gov even without a "us" param file?